### DISCUSSION TITLE
wildrift-gold-efficiency/5.1b/

### DISCUSSION THREAD

**Discussion by giscus** [changchiyou/wildrift-gold-efficiency#7] [Announcements]
# wildrift-gold-efficiency/5.1b/

Wild Rift Gold Efficiency of 5.1b

https://changchiyou.github.io/wildrift-gold-efficiency/5.1b/

**Comment by changchiyou** [comment]
[Gold Efficiency 5.1b：r/wildrift](https://www.reddit.com/r/wildrift/comments/1csp2sc/gold_efficiency_51b/)
